install_cuda.sh

```diff
@@ -0,0 +1,7 @@
+export LLAMA_CUBLAS=1
```

Comment:
so in your case :
```shell
set CMAKE_ARGS=-DLLAMA_CUBLAS=on
set FORCE_CMAKE=1
```

are not needed?

Reply:
I think that flag are for windows. this script work with linux. I already tested it :)
Later i will check for windows